src/blocks/types.tsx

```diff
@@ -155,4 +155,13 @@ export const BlockTypes: { [key: string]: BlockType } = {
         icon: giphyIcon,
         deprecated: true
     },
-};
\ No newline at end of file
+  "PixelArt": { 
+        type: "PixelArt",
+        displayName: "Pixel Art",
+        displayDescription: "A block to make then display pixel art",
+        emptyTitle: "Empty Pixel Art Block",
+        emptySubtitle: "Tap here to setup your Pixel Art block!",
+        icon: "PixelArtIcon", // TODO: insert your block icon here
```

Comment:
Don't forget to find an icon for your block! You'll need to add your icon here, as it won't compile without it.

Reply:
What is the best way to find / make icons? What are the expected dimensions of the icons?